### PR TITLE
[Keyboard] Dweebsmini split ortho

### PR DESCRIPTION
**Pull request by doesntfazer**
I made a new keyboard that I will be selling soon, Would like to add the firmware to the main branch

## Description
Split Ortho Keyboard.

## Types of Changes
Added a new keyboard branch

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/dweebsmini/rules.mk [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+#     differ from F_CPU if prescaling is used on the latter, and is required as the
+#     raw input clock is fed directly to the PLL sections of the AVR for high speed
+#     clock generation for the USB and other AVR subsections. Do NOT tack on a 'UL'
+#     at the end, this will be done automatically to create a 32-bit value in your
+#     source code.
+#
+#     If no clock division is performed on the input clock inside the AVR (via the
+#     CPU clock adjust registers or the clock division fuses), this will be equal to F_CPU.
+F_USB = $(F_CPU)
+
```
Comment (drashna) [inline keyboards/dweebsmini/rules.mk]: This block and be removed, actually.   

These are defined by default based on the MCU setting.

File: keyboards/dweebsmini/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _COLEMAK 1
+#define _DVORAK 2
+#define _LOWER 3
+#define _RAISE 4
+#define _ADJUST 16
```
Comment (fauxpark) [inline keyboards/dweebsmini/keymaps/default/keymap.c]: Non-consecutive layer numbers will generate extra empty layers in between, unnecessarily increasing flash usage. The better way to define layers is with an enum, eliminating the need to number them manually:

```c
enum layer_names {
    _QWERTY,
    _COLEMAK,
    _DVORAK,
    _LOWER,
    _RAISE,
    _ADJUST
};
```

File: keyboards/dweebsmini/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+#define ADJUST MO(_ADJUST)
+#define CTLESC CTL_T(KC_ESC)
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+[_QWERTY] = LAYOUT(
+   KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,  KC_LPRN,           KC_RPRN,  KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSPC, \
+   CTLESC,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,  KC_PGUP,           KC_PGDN,  KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, \
+   KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                               KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, \
+                              ADJUST,  KC_LGUI, RAISE, KC_SPACE,          KC_ENTER, LOWER,   KC_RALT,  KC_RCTL  \
+),
```
Comment (fauxpark) [inline keyboards/dweebsmini/keymaps/default/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

File: keyboards/dweebsmini/rules.mk (hunk 1 above)
Comment (fauxpark) [reply keyboards/dweebsmini/rules.mk]: Lines 45-47, too.

File: keyboards/dweebsmini/rules.mk [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   atmega32a    bootloadHID
```
Comment (fauxpark) [inline keyboards/dweebsmini/rules.mk]:
```suggestion
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
```

File: keyboards/dweebsmini/rules.mk (hunk 1 above)
Comment (doesntfazer) [reply keyboards/dweebsmini/rules.mk]: Fixed and resubmitted

File: keyboards/dweebsmini/keymaps/default/keymap.c (hunk 2 above)
Comment (doesntfazer) [reply keyboards/dweebsmini/keymaps/default/keymap.c]: Fixed and resubmitted

File: keyboards/dweebsmini/info.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "",
```
Comment (yanfali) [inline keyboards/dweebsmini/info.json]: missing a keyboard name

File: keyboards/dweebsmini/info.json [hunk 6]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "", 
+    "url": "", 
+    "maintainer": "qmk",
```
Comment (yanfali) [inline keyboards/dweebsmini/info.json]: needs a maintainer

File: keyboards/dweebsmini/keymaps/default/config.h [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+ // place overrides here
+#define DEFAULT_LAYER_SONGS { SONG(QWERTY_SOUND), \
```
Comment (yanfali) [inline keyboards/dweebsmini/keymaps/default/config.h]: you don't have audio, remove all of this

File: keyboards/dweebsmini/rules.mk [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+BOOTLOADER = caterina
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
```
Comment (yanfali) [inline keyboards/dweebsmini/rules.mk]: we recommend bootmagic lite

File: keyboards/dweebsmini/config.h [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0x0000
```
Comment (yanfali) [inline keyboards/dweebsmini/config.h]: pick a more unique value than this please

File: keyboards/dweebsmini/config.h [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+#define DEVICE_VER 0x0001
+#define MANUFACTURER KeyboardDweebs
+#define PRODUCT dweebsmini
+#define DESCRIPTION A custom keyboard
```
Comment (yanfali) [inline keyboards/dweebsmini/config.h]: a custom split 40% maybe?

File: keyboards/dweebsmini/config.h [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+/* serial.c configuration for split keyboard */
+#define SOFT_SERIAL_PIN D0
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* ws2812 RGB LED */
+#define RGB_DI_PIN D3
```
Comment (yanfali) [inline keyboards/dweebsmini/config.h]: you have no rgb, maybe delete this and the following lines

File: keyboards/dweebsmini/config.h [hunk 12]
```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 KeyboardDweebs
```
Comment (yanfali) [inline keyboards/dweebsmini/config.h]: Your copyright notices might need a legit identity to pass muster

File: keyboards/dweebsmini/readme.md [hunk 13]
```diff
@@ -0,0 +1,15 @@
+# dweebsmini
+
+![dweebsmini](https://i.redd.it/7oa8lp2q6mr31.jpg)
+
+This is a split Ortho keyboard, inspired by the CRKBD and the ErgoDash Mini
+
+Keyboard Maintainer: [KeyboardDweebs](HTTPS://KeyboardDweebs.com)
```
Comment (yanfali) [inline keyboards/dweebsmini/readme.md]: put extra space or <br> behind the next 3 lines or the document won't render correctly. Thanks

File: keyboards/dweebsmini/keymaps/default/keymap.c [hunk 14]
```diff
@@ -0,0 +1,88 @@
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum layer_names {
+    _QWERTY,
+    _COLEMAK,
```
Comment (yanfali) [inline keyboards/dweebsmini/keymaps/default/keymap.c]: Please delete COLEMAK and DVORAK as you're not using them in keymap at all.

File: keyboards/dweebsmini/keymaps/default/keymap.c (hunk 3 above)
Comment (doesntfazer) [reply keyboards/dweebsmini/keymaps/default/keymap.c:33]: I am leaving it the way it is

**Pull request closed without merging**